modules/statistics-provider-influx/src/main/java/org/opencastproject/statistics/provider/influx/StatisticsProviderInfluxService.java

```diff
@@ -170,4 +193,49 @@ private void disconnectInflux() {
     }
   }
 
+  @Override
+  public void writeDuration(
```

Comment:
Wouldn't the StatisticsWriter warrant its own class class like StatisticsWriterImpl, especially if this gets extended in the future?

Reply:
I get your point, but I'd like to file this under "premature abstraction". As soon as this gets extended, we'll see how this class is supposed to look like.